helm/olmv1/values.yaml

```diff
@@ -8,6 +8,7 @@ options:
     enabled: true
     deployment:
       image: quay.io/operator-framework/operator-controller:devel
+      replicas: 2
```

Comment:
I think the default should remain `1`, and set `podDisruptionBudget.enabled` to `false` as default as well. ` >1` is required for HA setups. We should not implicitly enforce after an upgrade that OLM suddenly consumes more cluster resources (i.e. pods). Such things should not hidden from users.